### ISSUE TITLE
[Non-coding] Demo Slides

### ISSUE THREAD

**Issue by Jason-KChen** [JKChenFZ/Minerva#106]
## Why/What
- Final Demo slides for BDS class
- Requirements are detailed below
- No limit on the number of participants per section as long as topics are not in collision
- Audio narrative should be dropped in `CS6220 Demo Audio` folder
- Template Slide deck has been created, see `CS6220 Demo Slides` under the project folder

## Guideline
1. Demo ppt should include the following design and development highlights
> Your project system architecture: Including components developed by you and open source modules (including pre-trained models), marking clearly the code modules you add to the open source modules or pre-trained models you use, including hyperparameter optimization, GUI, input data preprocessing/cleaning, output data visualization, etc.

- Overall architecture (+ Bert): @JKChenFZ 
- __Add Your Name Here__
- __Add Your Name Here__

> A use case example to walk through your project prototype

- __Add Your Name Here__
- __Add Your Name Here__

> A diagram to illustrate your model training process, including any tuning/validation step if used.

- __Add Your Name Here__
- __Add Your Name Here__

2. Demo Scenarios
> Demo ppt should include your demo scenarios:
a) easy cases that work for your approach and existing approaches you compared or intuitive baseline you compared;
b) good case that your approach outperform baseline or existing approach; elaborate why.
c) challenging scenarios which your approach cannot handle. Discuss how existing approaches handle them and elaborate how your approach might be improved to address the challenges identified.

- Bert vs various statistical approaches: @JKChenFZ 
- __Add Your Name Here__

3. Walk-through component
> Demo ppt should include a code walkthrough component: select two pieces of your code components to show the logic (annotated screen dump) and the working process, such as input, runtime screen shot and output. For example, training code walkthrough can show your training hyperparameter configuration and run time output; and the model prediction component walk through can show the input, the prediction time, accuracy and confidence and prediction result. 

- Bert in action: @JKChenFZ 
- __Add Your Name Here__

4. Experiemental results
> Each project should show three experimental measurement results (in figure or table), demonstrating your prototype in terms of accuracy, training time, prediction time comparison for at least 4 different configurations of your prototype: such as different sizes of datasets, different configuration of hyperparameters, different models and algorithms used, or different datasets. Provide your analysis and elaboration on the results

- Bert tuning and comparison: @JKChenFZ 
- __Add Your Name Here__

**Comment by Jason-KChen** [comment]
pinging @MrNickWan @Yuliy97 for exposure

**Comment by Jason-KChen** [comment]
### Important Update:

- A structural change has been applied to the demo slide deck. All slides from the workshop slides has been copied/imported into the demo slide deck. All existing slides in the demo slide deck has been pushed to the end of the whole deck deck. A partition slide has been added (named `DIVIDER_WILL_DEL`) to separated what has been copied and what was there originally. At the time of writing this update, this divider slide is the 24th slide
- The aim of the change is to facility slide re-use (This is explicitly allowed, with the final slide deck can be the exact same as the demo slides)

## What to do next?
1. Review all existing slides tagged with your name (including intro/outtro). If you believe your slide is still a good fit going forward, do the following two things. (1) Add `Workshop` after the speaker name to indicate slide re-use (as per submission requirement). (2) Copy the corresponding voice-over file from your workshop audio folder into the `CS6220 Demo Audio` folder. (3) Make minor edits to the re-used slides if you need to, consider doing a fresh voice-over if the changes are big.
2. Navigate to the end of the slide deck (all slides after the divider slide), move/inject your slides (slides with your names) into the correct position before the divider slide.
3. Make the corresponding voice-over and drop the file(s) into `CS6220 Demo Audio`


__Note__:
1. Voice-over file should have the exact same name as the slide title in all lower case (e.g. this_is_a_slide_name_workshop). Replace apostrophe and `-` with `_` in your naming to facilitate the final process
2. A video will be generated approximately `on 11/25 at 1:30 AM` (with grace period. Based on my understanding, the grace period for demo ends `on 11/25 at 9 AM` and the grace period for the final report ends `on 11/30 at 9 AM`).
3. Generated video will be uploaded and the corresponding link will be added to the first page of the slide
4. Video will not be submitted (will be over canvas limit), only the slide deck right after the same time above

**Comment by Jason-KChen** [comment]
Pinging stakeholders @Yuliy97 @MrNickWan  on important update above

**Comment by Yuliy97** [comment]
Finished slides and uploaded audios.

**Comment by Jason-KChen** [comment]
@Yuliy97 There are a few more things left.

1. Are the audio files in the orphan folder corrected? if so, please remove them
2. Please tag each of your assigned slide to either `demo` or `workshop`
3. For the workshop ones, are the audio being imported? I can't tell based on your naming convention. Please add the suffix. `demo` / `workshop`

**Comment by Yuliy97** [comment]
Done.

**Comment by Jason-KChen** [comment]
Submitted
